numba_dpex/core/utils/kernel_launcher.py

```diff
@@ -11,6 +17,48 @@
 from numba_dpex.dpctl_iface import libsyclinterface_bindings as sycl
 from numba_dpex.dpctl_iface._helpers import numba_type_to_dpctl_typenum
 
+MAX_SIZE_OF_SYCL_RANGE = 3
```

Comment:
There is a place in `parfor_lowerer.py` where the same hard coing is done. It will be goo to move the global to `dpctl_iface` and use it from there.